src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -3070,490 +3115,551 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
-                SetupOutputVariable(state,
-                                    "Zone Mixing Sensible Heat Gain Energy",
-                                    OutputProcessor::Unit::J,
-                                    state.dataHeatBal->ZnAirRpt(state.dataHeatBal->CrossMixing(Loop).FromZone).MixHeatGain,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    state.dataHeatBal->Zone(state.dataHeatBal->CrossMixing(Loop).FromZone).Name);
-                SetupOutputVariable(state,
-                                    "Zone Mixing Latent Heat Loss Energy",
-                                    OutputProcessor::Unit::J,
-                                    state.dataHeatBal->ZnAirRpt(state.dataHeatBal->CrossMixing(Loop).FromZone).MixLatentLoss,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    state.dataHeatBal->Zone(state.dataHeatBal->CrossMixing(Loop).FromZone).Name);
-                SetupOutputVariable(state,
-                                    "Zone Mixing Latent Heat Gain Energy",
-                                    OutputProcessor::Unit::J,
-                                    state.dataHeatBal->ZnAirRpt(state.dataHeatBal->CrossMixing(Loop).FromZone).MixLatentGain,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    state.dataHeatBal->Zone(state.dataHeatBal->CrossMixing(Loop).FromZone).Name);
-                SetupOutputVariable(state,
-                                    "Zone Mixing Total Heat Loss Energy",
-                                    OutputProcessor::Unit::J,
-                                    state.dataHeatBal->ZnAirRpt(state.dataHeatBal->CrossMixing(Loop).FromZone).MixTotalLoss,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    state.dataHeatBal->Zone(state.dataHeatBal->CrossMixing(Loop).FromZone).Name);
-                SetupOutputVariable(state,
-                                    "Zone Mixing Total Heat Gain Energy",
-                                    OutputProcessor::Unit::J,
-                                    state.dataHeatBal->ZnAirRpt(state.dataHeatBal->CrossMixing(Loop).FromZone).MixTotalGain,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    state.dataHeatBal->Zone(state.dataHeatBal->CrossMixing(Loop).FromZone).Name);
+        assert(mixingNum == state.dataHeatBal->TotCrossMixing);
```

Comment:
Does the `assert` failing imply a user error or a developer error? `assert`s are for the latter.

Reply:
Developer error. Just checking here to be sure.